docs/source/annotated_examples.rst

```diff
@@ -441,6 +386,34 @@ source: `scrollbar.py <https://github.com/enthought/chaco/tree/master/examples/d
 
 .. image:: example_images/scrollbar.png
 
+``simple_line.py``
+------------------
+Draws several overlapping line plots.
+
+Double-clicking on line or scatter plots opens a Traits editor for the plot.
+
+source: `simple_line.py <https://github.com/enthought/chaco/tree/master/examples/demo/simple_line.py>`_
+
+.. image:: ../images/simple_line.png
+
+.. [COMMENT]::
+
+    ``simple_polar.py``
```

Comment:
not sure why this is commented out

Reply:
Im not sure either... A search through the issues for "polar" only yields 2 results.  Possibly because of #47 ?

I am not sure, I'm happy to leave it or un-comment it

Reply:
Im gonna go ahead an push this PR through as is, we can always come back to uncommenting this if need be